org.OpenT2T.Sample.SuperPopular.Thermostat/Wink Thermostat/js/thingTranslator.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+var https = require('https');
+var wh = require("opent2t-translator-helper-wink"); //link locally for now
+
+// logs device state
+function logDeviceState(device) {
+    if (typeof (device) !== 'undefined') {
+        console.log('  device.name          : ' + device.name);
+        console.log('  device.props         : ' + device.props);
+    } else {
+        console.log('device is undefined');
+    }
+};
+
+var deviceId, accessToken;
+
+// module exports, implementing the schema
+module.exports = {
+
+    device: null,
```

Comment:
Delete this field. (I realize you copied this from somewhere else.) All the `this.device` references below don't even reference it anyway when running in a JXCore environment, because JXCore doesn't set `this` when calling the methods, so it defaults to the global object... So it happens to work, but it doesn't actually use this field.

I'm not sure you even need any references to `this.device` anyway, because `logDeviceState` doesn't seem to serve a useful purpose.